src/main/java/uk/gov/companieshouse/api/strikeoffobjections/model/request/CreatedBy.java

```diff
@@ -0,0 +1,22 @@
+package uk.gov.companieshouse.api.strikeoffobjections.model.request;
+
+public class CreatedBy {
+    private String id;
+    private String emailAddress;
```

Comment:
just 'email'

Reply:
That's exactly what it was before.

Reply:
good

Reply:
Not good.

Reply:
it's what is used in other CH collections like accounts.users. We should get an official CH standard for email